locales/en-US/runtime.json

```diff
@@ -77,15 +82,52 @@
             "stopped": "Stopped",
             "missing-types": "Waiting for missing types to be registered:"
             
+        },
+        "Flow": {
+            "unknown-type": "Unknown type: __type__",
+            "missing-types": "missing types"
+        },
+        "index": {
+            "unrecognised-id": "Unrecognised id: __id__",
+            "type-in-use": "Type in use: __msg__",
+            "unrecognised-module": "Unrecognised module: __module__"
+        },
+        "registry": {
+            "loader": {
+                "already-loaded": "__file__ already loaded",
+                "file-not-exist": "Error: __file__ does not exist",
+                "already-registered": "__type__ already registered",
+                "settings-unavailable": "Settings unavailable",
+                "module-already-loaded": "Module already loaded"
+            },
+            "localfilesystem": {
+                "module-not-found": "Cannot find module '__module__'"
+            },
+            "registry": {
+                "settings-unavailable": "Settings unavailable",
```

Comment:
I renamed settings.settings-not-available to settings.not-available in runtime.json file then changed in /red/settings.js.  Also, I removed "loader" and "registry" blocks under "registry" because I did undo /red/nodes/loader.js and /red/nodes/registry.js files.